docs/html/conf.py

```diff
@@ -12,22 +12,30 @@
 sys.path.insert(0, docs_dir)
 
 # -- General configuration ------------------------------------------------------------
-
 extensions = [
-    # first-party extensions
-    "sphinx.ext.autodoc",
-    "sphinx.ext.todo",
-    "sphinx.ext.extlinks",
-    "sphinx.ext.intersphinx",
-    # our extensions
+    # extensions common to 'man' and 'html' builds
```

Comment:
I'd also recommend keeping the order of extensions: first-party -> third-party -> local. Just like with regular imports.